include/trackjoint/utilities.h

```diff
@@ -75,7 +75,7 @@ Eigen::VectorXd DiscreteDifferentiationWithFiltering(const Eigen::VectorXd& inpu
  * input input_vector any vector, such as position
  * return a vector of normalized values between 0 and 1
  */
-Eigen::VectorXd normalize(const Eigen::VectorXd& x);
+Eigen::VectorXd Normalize(const Eigen::VectorXd& x);
```

Comment:
Yikes, I guess none of these functions should be capitalized. Would you mind making all of the camelCase?